### PR TITLE
🦺 Add DTO validation for book-author endpoints group

### PR DESCRIPTION
**Pull request by TellFruit**

### CODE REVIEW

File: backend/Wheelingful.API/Extensions/Validation/BeValidBase64Extension.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿namespace Wheelingful.API.Extensions.Validation;
+
+public static class BeValidBase64Extension
+{
+    public static bool BeValidBase64(this string value)
+    {
+        try
+        {
+            _ = Convert.FromBase64String(value);
```
Comment (VladislavAntonyuk) [inline backend/Wheelingful.API/Extensions/Validation/BeValidBase64Extension.cs:9]: there should be a method Convert.TryFromBase64String

File: backend/Wheelingful.API/Program.cs [hunk 2]
```diff
@@ -30,12 +31,15 @@
     .AddRoles<IdentityRole>()
     .AddEntityFrameworkStores<WheelingfulDbContext>();
 
-builder.Services.AddCoreServices();
-builder.Services.AddApiServices();
-
 builder.Services.AddOptions();
 builder.Services.AddCoreOptions(builder.Configuration);
 
+builder.Services.AddFluentValidationAutoValidation();
```
Comment (VladislavAntonyuk) [inline backend/Wheelingful.API/Program.cs]: I saw you added it for the book-author endpoints and you added it here as well. do we need it in 2 places?

File: backend/Wheelingful.API/Wheelingful.API.csproj [hunk 3]
```diff
@@ -8,10 +8,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="FluentValidation" Version="11.9.0" />
```
Comment (VladislavAntonyuk) [inline backend/Wheelingful.API/Wheelingful.API.csproj]: I suppose you don't need to add this package explicitly.

**Review by VladislavAntonyuk — APPROVED** [review]

File: backend/Wheelingful.API/Extensions/Validation/BeValidBase64Extension.cs (hunk 1 above)
Comment (TellFruit) [reply backend/Wheelingful.API/Extensions/Validation/BeValidBase64Extension.cs:9]: There is one, but that would be no better. It requires to write in variables I'm not going to use:
public static bool TryFromBase64String (string s, Span<byte> bytes, out int bytesWritten);

File: backend/Wheelingful.API/Program.cs (hunk 2 above)
Comment (TellFruit) [reply backend/Wheelingful.API/Program.cs]: It was implied to be necessary on the repository page:
image
But in fact it is not. Now that you mentioned it, I tried to remove it and the validation was functional.

File: backend/Wheelingful.API/Wheelingful.API.csproj (hunk 3 above)
Comment (TellFruit) [reply backend/Wheelingful.API/Wheelingful.API.csproj]: Good point. It is referenced by the other lib.

**Pull request merged**